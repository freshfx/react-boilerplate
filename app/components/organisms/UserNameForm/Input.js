import styled from '@emotion/styled'

const Input = styled.input`
  outline: none;
  border: none;
  border-bottom: 1px dotted #999;
  background-color: transparent;
`

export default Input
