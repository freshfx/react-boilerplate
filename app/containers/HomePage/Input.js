import styled from '@emotion/styled'

const Input = styled.input`
  outline: none;
  border-bottom: 1px dotted #999;
`

export default Input
