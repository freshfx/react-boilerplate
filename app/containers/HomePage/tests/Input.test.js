import React from 'react'
import {render} from '@testing-library/react'

import Input from '../Input'

describe('Input', () => {
  it('should match snapshot', () => {
    const {container} = render(<Input />)
    expect(container.firstChild).toMatchSnapshot()
  })
})

