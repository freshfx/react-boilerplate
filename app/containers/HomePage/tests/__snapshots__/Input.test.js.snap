// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> should match snapshot 1`] = `
.c0 {
  outline: none;
  border-bottom: 1px dotted #999;
}

<Input>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "Input-sc-19rkjmi-0",
          "isStatic": false,
          "lastClassName": "c0",
          "rules": Array [
            "outline:none;border-bottom:1px dotted #999;",
          ],
        },
        "displayName": "Input",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "Input-sc-19rkjmi-0",
        "target": "input",
        "toString": [Function],
        "usesTheme": false,
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <input
      className="c0"
    />
  </StyledComponent>
</Input>
`;
